Sources/Sentry/SentryTracer.m

```diff
@@ -223,7 +223,7 @@ - (void)removeTagForKey:(NSString *)key
 
 - (void)finish
 {
-    [self finishWithStatus:kSentrySpanStatusUndefined];
+    [self finishWithStatus:kSentrySpanStatusOk];
```

Comment:
`m`: It's a bad sign, that no test needs to be modified for this change. Can we please add one?